release/Dockerfile

```diff
@@ -18,6 +18,6 @@ COPY tokenizer.go       ./tokenizers/
 COPY tokenizers.h       ./tokenizers/
 COPY --from=builder-rust \
     /workspace/target/release/libtokenizers.a \
-    ./tokenizers/
+    ./tokenizers/lib/${DOCKER_TARGETPLATFORM}/
 COPY ./test/data        ./test/data
-RUN go run .
+RUN go run -ldflags="-extldflags '-L./tokenizers/lib/${DOCKER_TARGETPLATFORM}'" .
```

Comment:
We now need to specify the loading path as an ld flag at runtime since we no longer tell CGO where to find it at build time